### PR TITLE
CLUS editing

### PR DESCRIPTION
**Pull request by ReneeBornstein**
* added some periods
* moved the promt links up to the
  sentences and added a period to the
  end of the sentences.
* added some commas for the long
  sentences.

### CODE REVIEW

File: labs/ucsm-powertool-101/2.md [hunk 1]
```diff
@@ -3,33 +3,30 @@
...
 
   <!--![](assets/images/ucsm-powertool-101-06.jpg)<br/><br/>-->
 
-  There is a lot of output from this Cmdlet, but perhaps we are only interested in seeing a specific attribute of the compute resource. Let's retrieve the **Dn**</br></br>
+  There is a lot of output from this Cmdlet, but perhaps we are only interested in seeing a specific attribute of the compute resource. Let's retrieve the **Dn**.</br></br>
 
-  3. To retrieve just the Dn of the compute resource, at the prompt type
 
-  `Get-UcsBlade | Select-Object Dn`
+  3. To retrieve just the Dn of the compute resource, at the prompt type do `Get-UcsBlade | Select-Object Dn`.
```
Comment (annegentle) [inline labs/ucsm-powertool-101/2.md]: "type do" could simply be "type" here -- looks that way in other similar sentences.

File: labs/ucsm-powertool-101/1.md [hunk 2]
```diff
@@ -11,11 +11,11 @@ All UCS objects are described in an XML schema, this schema defines the objects
...
 
-  - How to Connect to UCS Manager using UCS PowerTool
-  - How to Query UCS Manager objects
-  - How to Disconnect from UCS Manager
+  - How to connect to UCS Manager using UCS PowerTool
+  - How to query UCS Manager objects
+  - How to disconnect from UCS Manager
 
-  UCS Manager Objects are representations of logical and physical entities such as compute blades, pools, polices, service profiles, organizations and more.
+UCS Manager Objects are representations of logical and physical entities such as compute blades, pools, polices, service profiles, organizations and more.
```
Comment (annegentle) [inline labs/ucsm-powertool-101/1.md]: In another patchset, ([link](https://github.com/CiscoDevNet/ucs-manager-powertool-learning-labs/pull/4/files)) you have a comma after organizations, I think that's correct here also.

File: labs/ucsm-powertool-101/2.md (hunk 1 above)
Comment (ReneeBornstein) [reply labs/ucsm-powertool-101/2.md]: Looks like you made this change. I no longer see it.

File: labs/ucsm-powertool-101/1.md (hunk 2 above)
Comment (ReneeBornstein) [reply labs/ucsm-powertool-101/1.md]: Sorry; thought I'd done it. I did make the change, and also removed the extra image links.

File: labs/ucsm-powertool-101/2.md (hunk 1 above)
Comment (ReneeBornstein) [reply labs/ucsm-powertool-101/2.md]: Making the change. In another doc you said to put a comma between prompt and type, so I did that as well. Will make it consistent during all changes.

**Pull request merged**